.github/workflows/ci.yml

```diff
@@ -21,7 +21,7 @@ jobs:
       - name: Print forge version
         run: forge --version
 
-      # Backwards compatibility checks.
+      # Backwards compatibility checks: 0.y.0 (or oldest), and releases with known issues are checked - do not remove those.
```

Comment:
I thought we added the 0.6.12 check because the idea was to test the oldest and newest version of each supported minor version, since there are sometimes breaking changes in between. So we had 0.6.2 and 0.6.12 (0.6.2 is when interface inheritance was introduced, which is why we didn't support 0.6.0). Then we had 0.7.0 and 0.7.6 for the same reason. 

But regardless, since these checks are cheap (fast), we may as well just keep it to be safe and lean towards being more conservative?